test/extended/util/client.go

```diff
@@ -137,9 +138,8 @@ func NewCLIWithoutNamespace(project string) *CLI {
 		adminConfigPath:  KubeConfigPath(),
 		withoutNamespace: true,
 	}
-	g.AfterEach(cli.TeardownProject)
-	g.AfterEach(cli.kubeFramework.AfterEach)
 	g.BeforeEach(cli.kubeFramework.BeforeEach)
+	g.AfterEach(cli.kubeFramework.AfterEach)
```

Comment:
Looking at the ginkgo docs(not very familiar with how this code works), It looks to me like this moves where the various blocks are being registered but doesn't actually change their order, do you instead want JustAfterEach for this line so that it is called before the TeardownProject, if that what your hoping to achieve ?

Reply:
Yes, that was the intent. Let me try that. Also, seems incorrect to have the BeforeEach() and AfterEach() for a given resource to be spread around in different methods like that.